stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -127,8 +127,8 @@ public String getBenchDescription() {
   @Override
   public void prepare() throws Exception {
     if (mBaseParameters.mCluster) {
-      // For cluster mode, this function is called once before the job is submitted to the job
-      // service. Nothing should be done, otherwise the bench will break.
+      // Create test directory /fuseIOStressBench before job submitted to job service.
+      Files.createDirectories(Paths.get(mParameters.mLocalPath, "fuseIOStressBench"));
```

Comment:
do we use `"fuseIOStressBench"` in other places? if yes, try to use an constant
otherwise for users like me, i will be confused why create such a directory and not knowing why this directory is used for